components/TARGET_PSA/services/crypto/COMPONENT_SPE/psa_crypto_access_control.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019, Arm Limited and affiliates
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef PSA_CRYPTO_ACCESS_CONTROL_H
+#define PSA_CRYPTO_ACCESS_CONTROL_H
+
+#include <stdint.h>
+
+#include "psa_crypto_core.h"
+#include "crypto_platform.h"
+
+void psa_crypto_access_control_init(void);
```

Comment:
these functions should contain documentation?

Reply:
I don't think so, this is internal only. These could have been implemented as static functions in psa_crypto_partition.c...

Reply:
we could use `\internal` or using _internal suffix in the header or at least a comment in this header file.

Even internal functions should have documentation, hidden  from public (using `#if !defined(DOXYGEN_ONLY)`).

Reply:
Added in 5ac58b84dfa06f1c6cd76293dd4f01a5511124a0